test/leafTest.js

```diff
@@ -0,0 +1,151 @@
+      const testData = [
+        {
+          address: "0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5",
+          creationBlock: BigNumber.from("0xDFD871"),
+          id: 0x1995b5,
+          expectation:
+            "0x4ddc2d193948926d02f9b1fe9e1daa0718270ed50000000000DFD871001995B5",
+          //     read:    4ddc2d193948926d02f9b1fe9e1daa0718270ed5,0000000000DFD871,001995B5
+          //                    address                             creationBlock     id
+        },
+        {
+          address: "0x176f3dab24a159341c0509bb36b833e7fdd0a132",
+          creationBlock: BigNumber.from("0x5E8EEE0"),
+          id: 0x8f573,
+          expectation:
+            "0x176f3dab24a159341c0509bb36b833e7fdd0a1320000000005E8EEE00008F573",
+          //     read:    176f3dab24a159341c0509bb36b833e7fdd0a132,0000000005E8EEE0,0008F573
+          //                    address                             creationBlock     id
+        },
+        {
+          address: "0x2b6ed29a95753c3ad948348e3e7b1a251080ffb9",
+          creationBlock: BigNumber.from("0x771D8DB"),
+          id: 0xca009,
+          expectation:
+            "0x2b6ed29a95753c3ad948348e3e7b1a251080ffb9000000000771D8DB000CA009",
+          //     read:    2b6ed29a95753c3ad948348e3e7b1a251080ffb9,000000000771D8DB,000CA009
+          //                    address                             creationBlock     id
+        },
+      ]
+
+      testData.forEach(async (test) => {
+        const leaf = await leafInstance.make(
+          test.address,
+          test.creationBlock,
+          test.id,
+        )
+        expect(leaf).to.equal(test.expectation)
+      })
+    })
+  })
+
+  describe("operator", async () => {
+    it("retrieves the operator from the leaf", async () => {
+      const testData = [
+        {
+          leaf: "0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED50000000000DFD871001995B5",
+          //       4Ddc2D193948926D02f9B1fE9e1daa0718270ED5,0000000000DFD871,001995B5
+          //              address                             creationBlock     id
+          operator: "0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5",
+        },
+        {
+          leaf: "0x176F3DAb24a159341c0509bB36B833E7fdd0a1320000000005E8EEE00008F573",
+          //       176F3DAb24a159341c0509bB36B833E7fdd0a132,0000000005E8EEE0,0008F573
+          //              address                             creationBlock     id
+          operator: "0x176F3DAb24a159341c0509bB36B833E7fdd0a132",
+        },
+        {
+          leaf: "0x2B6eD29A95753C3Ad948348e3e7b1A251080Ffb9000000000771D8DB000CA009",
+          //       2B6eD29A95753C3Ad948348e3e7b1A251080Ffb9,000000000771D8DB,000CA009
+          //              address                             creationBlock     id
+          operator: "0x2B6eD29A95753C3Ad948348e3e7b1A251080Ffb9",
+        },
+      ]
+
+      testData.forEach(async (test) => {
+        const leaf = await leafInstance.operator(BigNumber.from(test.leaf))
```

Comment:
I think we should rename this to be something more specific, like `const operator`

Reply:
[c4178bf](https://github.com/keep-network/sortition-pools/pull/173/commits/c4178bf973bf88a6c665ac46cb22ac6a8efdfd79)
[5e30dbc](https://github.com/keep-network/sortition-pools/pull/173/commits/5e30dbc5d12ae71dbd3284364f98c0c9d4b1af76)
[dab2e9e](https://github.com/keep-network/sortition-pools/pull/173/commits/dab2e9ec2bd0a74201771d170d5f00f1bb5b612d)